samples/cellular/nrf_cloud_multi_service/overlay-coap_nrf_provisioning.conf

```diff
@@ -0,0 +1,40 @@
+# Enable provisioning library/provisioning shell dependencies
```

Comment:
Should the HTTP version of the overlay be renamed to `http_nrf_provisioning`?

I also think docs should probably be updated to mention both overlays

Reply:
I haven't done the docs yet, but I agree.